src/app/objective/objective.component.ts

```diff
@@ -37,12 +37,24 @@ export class ObjectiveComponent implements OnInit {
   @Output() onMoveBucket = new EventEmitter<[ImmutableObjective, ImmutableObjective, ImmutableBucket]>();
   @Output() onDelete = new EventEmitter<ImmutableObjective>();
   @Output() onChanged = new EventEmitter<[ImmutableObjective, ImmutableObjective]>();
-  
+
   constructor(public dialog: MatDialog) { }
 
   ngOnInit() {
   }
 
+  hasPeopleAvailable() {
+    if (this.objective!.assignments.length) {
```

Comment:
Strictly speaking, we should probably check that the assignments have positive value, i.e. something like:

```typescript
if (this.objective!.assignments.find(a => a.commitment > 0) {
    return true;
}
```

Reply:
Fixed it, thanks!